rspec_event_list.rb

```diff
@@ -0,0 +1,53 @@
+require 'rspec/autorun'
+require_relative 'event_list'
+
+describe EventList do
+
+  let(:event) {Event.new("Battle of the Bands", "Downtown", "2021-09-02", "16:00")}
+  let(:event_list) {EventList.new}
+
+  it "adds an event" do
+  	event_list.add_event("2021-09-02", event)
+  	puts('Test Case 1')
+  	p(event_list)
+  	puts('Test Case Ended')
+  end
+  it "updates an event" do
+    puts('Test Case 2')
+    event_list.add_event("2021-09-02", event)
+    event_list.update_event("2021-09-02", "Battle of the Bands")
+
+  	p(event_list)
+  	puts('Test Case Ended')
+  end
+  it "deletes an event" do
```

Comment:
it would be nice we write test cases for failure. For example,  we have 2 events (1, 2) and user tries to delete event (3) which we do not have then expect event not exists test case